ui/ui-community/__tests__/integration/happyPath.ts

```diff
@@ -47,4 +47,14 @@ describe("integration", () => {
     },
     helper.mediumTime
   );
+
+  test(
+    "can render a page with a form",
+    async () => {
+      await browser.get(url + "/session/sign-in");
+      const header = await browser.findElement(by.tagName("h1")).getText();
+      expect(header).toContain("Sign In");
+    },
+    helper.mediumTime
+  );
```

Comment:
This would have previous failed without the `experimental-vm-modules` flags... now it is OK!